modules/mco-update-boot-images-disable.adoc

```diff
@@ -34,4 +34,4 @@ spec:
   managedBootImages: <1>
     machineManagers: []
 ----
-Remove the parameters  that are listed under `machineManagers` and add the `[]` characters to disable the updating of boot images.
+<1> Remove the parameters  that are listed under `machineManagers` and add the `[]` characters to disable the updating of boot images.
```

Comment:
```suggestion
<1> Remove the parameters listed under `machineManagers`, and add the `[]` characters to disable boot image updates.
```

Just a suggestion to remove passive voice from the sentence. Also, there's a double space between _parameters_ and _that_.

Reply:
```suggestion
<1> Remove the parameters listed under `machineManagers` and add the `[]` characters to disable boot image updates.
```

@shreyasiddhartha comma is not necessary here. Looks good overall.